src/comlink.ts

```diff
@@ -371,6 +375,39 @@ function throwIfProxyReleased(isReleased: boolean) {
   }
 }
 
+function releaseEndpoint(ep: Endpoint) {
+  return requestResponseMessage(ep, {
+    type: MessageType.RELEASE,
+  }).then(() => {
+    closeEndPoint(ep);
+  });
+}
+
+interface FinalizationRegistry<T> {
+  new (cb: (heldValue: T) => void): FinalizationRegistry<T>;
+  register(weakItem: any, heldValue: T, unregisterToken: any): void;
```

Comment:
If `weakItem` or `unregisterToken` are not objects, [`register` will throw](https://tc39.es/proposal-weakrefs/#sec-finalization-registry.prototype.register)
```suggestion
  register(weakItem: object, heldValue: T, unregisterToken?: object | undefined): void;
```